restrict_download_region/restrict_region.py

```diff
@@ -0,0 +1,153 @@
+import os
```

Comment:
I had the line limit set to 120 as we do in the synapse python client. May I ask why 80 characters? In my personal opinion the code is much harder to read and organize when the line limit is 80 since it abruptly forces words to the next line, especially with nested indents.

Reply:
i don't have a hard rule about this but i guess i have the opposite opinion that shorter lines make it easier to read and review.  I guess my eyes aren't great anymore so i keep my resolution a bit larger than you young'ins.  anyways if 120 works better for you then just keep as is.  if you feel like making it easier for the older folk then maybe consider around 100 chars?